data/hazards.txt

```diff
@@ -124,3 +124,20 @@ hazard "Rulei Gravity Field"
 
 effect "gravity well"
 	"sound" "gravity well"
+
+
+hazard "Core Sheltered Heat"
+	"duration" 1
+	"strength" 0.65 0.85
+	"range" 50000
+	weapon
+		"burn damage" 0.06
+		"relative heat damage" 0.0012
+#replace 0.0006 rel. heat damage with 0.00006 rel. burn damage if/when implemented.
+
+hazard "Core Shadow Passing"
```

Comment:
This is so that if a ship is disabled by the first hazard, it will receive periodic opportunities to recover and escape. But the variable hazard strength is too random for this to work without a second hazard - this setup produces a pulse of sufficient duration to actually be meaningful.

Reply:
This is the exact same set of hazards used for the Xapleaux systems, but they use them with rather more heat. So yes, it is needed.
This is just seeing the edge case of their implementation.

Reply:
The hazards are set to different frequencies in different systems so that we don't need 30 unique hazards in the broader implementation. In systems with higher frequency, the cooling off feature is a needed balance element, in systems with low frequency, it is not.

Reply:
IT IS NOT IN THE BLOODY SYNDICATE SYSTEM - WHICH YOU WOULD KNOW IF YOU LOOKED AT THE MAP FILE

Stop going around in circles on two different comment threads and mixing up the difference between the definition of a hazard, and the application of it.

This particular thread is over.